main.tf

```diff
@@ -156,7 +156,7 @@ resource "aws_glue_job" "sales_val_flagging" {
     "--rolling_window_num"        = 12
     "--time_frame_start"          = "2014-01-01"
     "--dev_bounds"                = "2,3"
-    "--additional-python-modules" = "boto3==1.28.12,pandas==2.0.2"
+    "--additional-python-modules" = "boto3==1.28.12,pandas==1.3.5,awswrangler==2.20.1,pyathena==2.25.2"
```

Comment:
These are the latest available packages available in the new spark configuration.